browser/js/common/directives/kit-fields/kit.code.field.controller.js

```diff
@@ -0,0 +1,12 @@
+app.directive('kitCodeField', function() {
+
+    return {
+        restrict: 'E',
+        templateUrl: 'js/common/directives/kit-fields/kit-code-field.html',
+        scope: { model: '=ngModel', question: '=' },
```

Comment:
I'd recommend against "generic" names like `model`. IMO it makes the code harder to talk about.